app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,6 @@
 <resources>
     <string name="app_name">RMMA</string>
-    <string name="hello_android">Hello Android! Version: %s</string>
+    <string name="catFact_1">Cat Fact #1: %s</string>
+    <string name="catFact_2">Cat Fact #2: %s</string>
+    <string name="catFact_3">Cat Fact #3: %s</string>
```

Comment:
It's not a good idea to add a new string for every fact. What about 500 facts? 
You can use `Cat Fact: #%n: %s` and use it like `Context#getString(1, "Fact 1")`